app/javascript/packs/AddNewRestPopup.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import $ from 'jquery';
+
+class AddNewRestPopup extends React.Component {
+
+    constructor() {
+        super();
+        this.state = {
+            name: "",
+            address: "",
+            tenbis: false,
+            max_delivery_time: 10,
+            cuisine: ""
+        };
+
+        this.updateChange = this.updateChange.bind(this);
+        this.newRestaurantSubmit = this.newRestaurantSubmit.bind(this);
+    }
+
+    updateChange(e) {
+        const event_id = e.target.id;
+        const event_value = e.target.value;
+
+        if (event_id === "restaurant_name") {
```

Comment:
I think it would be nicer to use a switch-case here. It's a perfect use-case here:
https://www.w3schools.com/jsref/jsref_switch.asp